File: CPP/string_character_frequency.cpp
// Character Frequency Count
#include<iostream>
using namespace std;
int main() {
    char str[100];
    int freq[256] = {0}; // assumption

    cout<<"enter string name : "<<endl;
    cin>>str;
    for(int i = 0; str[i] !='\0'; i++) {
        char ch = str[i];
        freq[ch]++;

    }

    for(int i = 0; str[i] !='\0'; i++) {
        char ch = str[i];
        if (freq[ch] != 0) {
            cout<<ch<< " - "<<freq[ch]<<endl;
            freq[ch] = 0;
        }

    }
    return 0;

}

File: Java_Practice/Python/string_character_frequency.py
#Character Frequency Count
# Take input from user
str_input = input("Enter a string: ")

# Create an empty dictionary to store frequency
freq = {}

# Count frequency
for ch in str_input:
    if ch in freq:
        freq[ch] += 1
    else:
        freq[ch] = 1

# Print frequency
for ch, count in freq.items():
    print(f"{ch} - {count}")
